08-More_About_ASDF/08-Solutions.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>It may be good to point out that the duplicate array is referenced, not copied.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/roman-data-workshop/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>